rabbitmq_messages_incoming_rate

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/python
+import commands
+import json
+import sys
+
+#you'll have to set those to match your setup
+MQ_USER = 'guest'
+MQ_PASS = 'guest'
```

Comment:
You should get those from the environment so that they can be securely configured from /etc/munin/plugin-conf.d/ using env.MQ_USER and env.MQ_PASS.